ui/css/design-system/typography.scss

```diff
@@ -1,80 +1,80 @@
 @use "sass:map";
 
-$fa-font-path: 'fonts/fontawesome';
+$fa-font-path: '~@fortawesome/fontawesome-free/webfonts';
```

Comment:
What's the idea here - is there some loader or plugin which will make this work? (reverting just this part seems to make it pass)